networkx/algorithms/centrality/eigenvector.py

```diff
@@ -42,7 +42,7 @@ def eigenvector_centrality(G, max_iter=100, tol=1.0e-6, nstart=None, weight=None
     weight : None or string, optional (default=None)
       If None, all edge weights are considered equal.
       Otherwise holds the name of the edge attribute used as weight.
-
+      In this measure the weight is interpreted as the connection strength.
```

Comment:
```suggestion
      In this measure the weight is interpreted as the connection strength.
      
```
There needs to be a blank line in between one section and the next section's header, otherwise the docstring won't be parsed properly.